src/fusor/translator.py

```diff
@@ -180,6 +193,37 @@ def _get_genomic_ac(self, chrom: str, build: Assembly) -> str:
             raise ValueError
         return alias_list[0].split(":")[1]
 
+    def _process_gene_symbols(
+        self, gene_5prime: str, gene_3prime: str, caller: Caller
+    ) -> dict[GeneElement | UnknownGeneElement]:
+        """Process gene symbols to create GeneElements or UnknownGeneElements
+
+        :param gene_5prime: The 5' gene symbol
+        :param gene_3prime: The 3' gene symbol
+        :param caller: The gene fusion caller
+        :return A dictionary of GeneElements or UnknownGeneElements
+        """
+        gene_5prime_element = gene_3prime_element = None
+        if gene_5prime == "NA":
+            gene_5prime_element = gene_5prime = UnknownGeneElement()
+            gene_5prime = None
+        if gene_3prime == "NA":
+            gene_3prime_element = gene_3prime = UnknownGeneElement()
+            gene_3prime = None
```

Comment:
Is this what you meant? It doesn't make sense to assign to UnknownGeneElement() if you're just going to assign it to none in the following line
```suggestion
        if gene_5prime == "NA":
            gene_5prime_element = UnknownGeneElement()
            gene_5prime = None
        if gene_3prime == "NA":
            gene_3prime_element = UnknownGeneElement()
            gene_3prime = None
```